client/src/pages/guide/english/cplusplus/stack/index.md

```diff
@@ -163,3 +163,14 @@ int main ()
     2
     1
     Out of loop
+##Uses of Stack
```

Comment:
Please Fix the heading, there should be a space b/w # and letter.

Reply:
fixed it.